### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by macasas** [RasaHQ/rasa_core#651]
I have a fresh installation of Ubuntu with Python 3.6.5 installed.  I am using pip3 to install Rasa, following the instructions on the nlu.rasa.com/installation.html page and then the core.rasa.com/installation.htmlpage

During the installation I get an Segmentation error in 2 places
pip3 install rasa_nlu completes without an error
but
pip install rasa_nlu[spacy] completes without an error, but then pops up a Segmentation fault (core dumped) message
then pip3 install rasa_core is doing the same thing

I have cross posted because each fault is related to different repos, apologies if this is wrong.
https://github.com/RasaHQ/rasa_nlu/issues/1178

Here is the end of my log from trying to install core.
  Stored in directory: /home/dave/.cache/pip/wheels/33/56/20/ebe49a5c612fffe1c5a632146b16596f9e64676768661e4e46
Successfully built slackclient jsonpickle ConfigArgParse networkx gast absl-py tzlocal itsdangerous pysocks docopt html5lib MarkupSafe
Installing collected packages: graphviz, pyyaml, chardet, certifi, idna, urllib3, requests, incremental, hyperlink, attrs, six, Automat, setuptools, zope.interface, constantly, Twisted, Werkzeug, klein, tqdm, jmespath, docutils, python-dateutil, botocore, s3transfer, boto3, future, jsonschema, numpy, pyparsing, packaging, pytz, cycler, kiwisolver, matplotlib, cloudpickle, typing, simplejson, greenlet, gevent, pathlib, rasa-nlu, websocket-client, slackclient, python-telegram-bot, fbmessenger, jsonpickle, mattermostwrapper, ruamel.yaml, scipy, h5py, keras-applications, keras-preprocessing, keras, gast, termcolor, astor, grpcio, protobuf, markdown, html5lib, bleach, wheel, tensorboard, absl-py, tensorflow, colorhash, ConfigArgParse, tzlocal, apscheduler, redis, fakeredis, MarkupSafe, Jinja2, click, itsdangerous, flask, decorator, networkx, PyJWT, pysocks, twilio, docopt, pykwalify, scikit-learn, humanfriendly, coloredlogs, flask-cors, rasa-core
Segmentation fault (core dumped)

**Comment by aashishg** [comment]
@macasas Did you try doing it with root user or a sudo user?

**Comment by macasas** [comment]
I sort of know what that means but I am not a programmer, hacker or network guru, I build conversations and personalities, I understand the technical side, the architecture and plumbing, I've even built bots using Javascript and the JOVO framework, but I want to drive Rasa without having to become a mechanic and know the intricacies under the hood. With Rasa I see a great opportunity to concentrate on the conversations and introduce a whole new set of people to bot building who can run an app, but are focused on the UX.  This is why I probably ask some dumb questions from time to time, and you will get "I don't know" answers to questions like this. The fact you are asking this question tells me something somewhere should be made simpler to achieve.  

If I remember correctly, I ran these commands with, without and using something like --client or --user. Same result in all cases.

Also, this is no longer the result since the new version was released. The latest upgrade did not show this error.
